.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/app-prototype
+    docker:
+      - image: circleci/ruby:2.4.2-node-browsers
+        environment:
+          PGHOST: 127.0.0.1
+          PGUSER: postgres
+          RAILS_ENV: test
+      - image: circleci/postgres:9.6.5-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: app_prototype_test
+          POSTGRES_PASSWORD: ""
+    steps:
+      - checkout
+
+      # Restore bundle cache
+      - type: cache-restore
+        key: bundle-{{ arch }}
+
+      # Bundle install dependencies
+      - run: bundle install --path vendor/bundle --jobs=4
+      - run: bundle clean --force
+
+      # Store bundle cache
+      - type: cache-save
+        key: bundle-{{ arch }}
+        paths:
+          - vendor/bundle
+
+      # Database setup
+      - run: bundle exec rake db:create
+      - run: bundle exec rake db:schema:load
+
+      # Run rspec in parallel
+      - type: shell
```

Comment:
how come this command uses a different syntax than the other shell commands (they use `run`)? is it a yaml formatting thing?

Reply:
No idea really. I copied it from the CCI 2 docs. I'll change it and test that it still works.

Reply:
Their examples mix and match the two. I think both are viable.

Reply:
yeah i wouldn't be surprised if they don't use a uniform syntax in their documentation